src/mon/Session.h

```diff
@@ -141,6 +130,23 @@ struct MonSession : public RefCountedObject {
   }
 };
 
+struct Subscription {
+  MonSession *session;
+  std::string type;
+  xlist<Subscription*>::item type_item;
+  version_t next;
+  bool onetime;
+  bool incremental_onetime;  // has CEPH_FEATURE_INCSUBOSDMAP
+
+  Subscription(MonSession *s, const std::string& t) : session(s), type(t), type_item(this),
+                                                 next(0), onetime(false), incremental_onetime(false) {}
+
+  void dump(ceph::Formatter *f) const {
+    f->dump_stream("name") << session->name;
+    f->dump_stream("entity_name") << session->entity_name;
```

Comment:
```suggestion
    f->dump_object("entity_name", session->entity_name);
```

?

Reply:
entity_name is a `EntityName` object, EntityName have no `dump` function, but it overload `operator<<`. `session->name` also same.